src/installer/image/installer.go

```diff
@@ -79,10 +127,6 @@ func (installer *Installer) InstallAgent(targetDir string) (bool, error) {
 	return true, nil
 }
 
-func (installer Installer) Cleanup() error {
-	return installer.props.DockerConfig.Cleanup(afero.Afero{Fs: installer.fs})
```

Comment:
you should call this in [src/controllers/csi/provisioner/install.go](https://github.com/Dynatrace/dynatrace-operator/pull/1996/files#diff-b6a60fe4f1b2d69b8d567c8eae450795fa8ff2259d85f3545f6b77fb1fc60d7f)
so the local files will be cleaned up

Reply:
@0sewa0 fixed in 01d083d1